framework/test/gtest/CMakeLists.txt

```diff
@@ -154,7 +154,7 @@ set_property(TEST ${us_gtest_test_exe_name} PROPERTY LABELS regular)
 
 # Run the GTest EXE from valgrind
 if(US_MEMCHECK_COMMAND)
-  add_test(NAME memcheck_${us_gtest_test_exe_name} COMMAND ${US_MEMCHECK_COMMAND} --error-exitcode=1 ${US_RUNTIME_OUTPUT_DIRECTORY}/${us_gtest_test_exe_name}
+  add_test(NAME memcheck_${us_gtest_test_exe_name} COMMAND ${US_MEMCHECK_COMMAND} --max-threads=2000 --error-exitcode=1 ${US_RUNTIME_OUTPUT_DIRECTORY}/${us_gtest_test_exe_name}
```

Comment:
the max-threads used for DS/ConfigAdmin tests is 1000. any reason for making this higher?